src/haddock/libs/libplots.py

```diff
@@ -633,22 +636,114 @@ def find_best_struct(ss_file, number_of_struct=10):
         )
     best_struct_df.reset_index(inplace=True)
     best_struct_df.rename(columns={"cluster-id": "Cluster ID"}, inplace=True)
-    return best_struct_df
 
+    # unclustered id is "-", it is replaced by "Unclustered"
+    best_struct_df["Cluster ID"] = best_struct_df["Cluster ID"].apply(
+        _fix_uncluster_name
+        )
+    return best_struct_df
 
-def _add_links(best_struct_df):
-    def format_cell(row):
-        # TODO add Download link
-        # TODO add PDB viewer link
-        # plotly renders a tag as text not as HTML,
-        # need to switch to Dash+markdown or make table html ourselves
-        # name = Path(row).name
-        # return f'<a href="../{row}" target="_blank" rel="noreferer">{name}</a>' # noqa: E501
-        return Path(row).name
 
-    table_df = best_struct_df.copy()
+def _ngl_viewer():
+    # http://nglviewer.org/ngl/gallery/
+    ngl_script = ("""
+        <script src="https://cdn.rawgit.com/arose/ngl/v2.1.0/dist/ngl.js">
+        </script>
+        <script>
+            var dialog;
+            var stage;
+
+            document.addEventListener("DOMContentLoaded", function () {
+                // Setup to load data from rawgit
+                NGL.DatasourceRegistry.add(
+                    "data",
+                    new NGL.StaticDatasource(
+                        "//cdn.rawgit.com/arose/ngl/v2.1.0/data/"
+                        )
+                );
```

Comment:
I think this code does nothing. As we are loading files from the local http server. Can it be removed?

```suggestion
```